serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc

```diff
@@ -1,107 +1,584 @@
-That said, the usage of in-memory Job Service can only be done for testing or development, but not for production.
+If the project is not configured to use the Job Service, all time-based actions will use an in-memory implementation of that service.
+However, this setup must not be used in production, since every time the application is restarted, all the timers are lost.
+This last is not suited for serverless architectures, where the applications might scale to zero at any time, etc.
 ====
 
-The main goal of the Job Service is to work with only active jobs. The Job Service tracks only the jobs that are scheduled and that need to be executed. When a job reaches a final state, the job is removed from the Job Service.
+== Jobs life-span
 
-When configured in your environment, all the jobs information and status changes are sent to the {product_name} `Data
+Since the main goal of the Job Service is to work with the active jobs, such as the scheduled jobs that needs to be executed, when a job reaches a final state, it is removed from the Job Service.
+However, in cases where you want to keep the information about the jobs in a permanent repository, you can configure the Job Service to produce status change events, that can be collected by the {product_name} `Data
 Index Service`, where they can be indexed and made available by GraphQL queries.
 
+== Executing
+
+To execute the Job Service in your docker or kubernetes environment, you must use the following image:
+
+* link:{jobs_service_image_url}[kogito-jobs-service-allinone]
+
+In the next topics you can see the different configuration parameters that you must use, for example, to configure the <<job-service-persistence,persistence mechanism>>, the <<job-service-eventing-api, eventing system>>, etc.
+More information on this image can be found link:{jobs_service_image_usage_url}[here].
+
+We recommend that you follow this procedure:
+
+1. Identify the <<job-service-persistence,persistence mechanism>> to use and see the required configuration parameters.
+2. Identify if the <<job-service-eventing-api, Eventing API>> is required for your needs and see the required configuration parameters.
+3. Identify if the project containing your workflows is configured with the appropriate xref:job-services/quarkus-extensions.adoc#job-service-quarkus-extensions[Job Service Quarkus Extension].
+
+Finally, to run the image, you must use the <<exposed-environment-variables, environment variables exposed by the image>>, and other configurations that you can set using <<using-environent-variables, additional environment variables>> or using <<using-java-like-system-properties, system properties with java like names>>.
+
+[#exposed-environment-variables]
+=== Exposed environment variables
```

Comment:
no `ENABLE_EVENTS` env var ?
See https://github.com/kiegroup/kogito-images/blob/main/kogito-jobs-service-allinone-image.yaml#L26

Reply:
I'm trying to not introduce this variable here, because it's particular to kafka, and now, we have http and kafka events.
So I thing to promote the usage of QUARKUS_PROFILE=kafka-events-support is better.
However, the old profile events-support still works for backward compatibility, so "old" installations (not sure if there are any) that might use the ENABLE_EVENTS var (even less probalby) will till work.

The other alternative is so review this "ENABLE_EVENTS" parameter, we can talk in the meeting.

Reply:
Maybe create an issue to review that one on images.